.github/workflows/build-natives.yaml

```diff
@@ -36,17 +36,21 @@ jobs:
       - if: matrix.os == 'windows-latest'
         name: Install Full OpenSSL Library
         run: |
-          choco install --no-progress openssl
+          vcpkg install openssl:x64-windows
       - if: matrix.os != 'windows-latest'
         name: Build with Maven
-        run: mvn -B package --file pom.xml
+        run: |
+          openssl version -v
+          mvn -B package --file pom.xml
       - if: matrix.os == 'windows-latest'
         name: Build with Microsoft Visual Studio native tools command prompt and Maven
         env:
           VCVARS: ${{ matrix.vcvars }}
         shell: cmd
         run: |
-          call "%VCVARS%"
+          call "C:\Program Files\Microsoft Visual Studio\2022\Enterprise\VC\Auxiliary\Build\vcvarsall.bat" x64          
```

Comment:
Just curious, is there a way to avoid duplicating the path here? Can we use `call "%VCVARS%"
 x64` since the env variable `VCVARS` was set to `matrix.vcvars` above?

Reply:
@fjuma Right! Fixed, btw here is the test tag that will have archive: https://github.com/Skyllarr/wildfly-openssl-natives/actions/runs/3686048703

Reply:
@fjuma Sorry I am not sure, is this what you mean https://github.com/Skyllarr/wildfly-openssl-natives/blob/6fa40e91b72eb06b0b849ddc322e1c4be5697cf8/.github/workflows/build-natives.yaml#L66-L68 ?

Reply:
The tests for wildfly-openssl are outside of this natives repo. So I was thinking something like this:

git clone https://github.com/wildfly-security/wildfly-openssl.git
cd wildfly-openssl
mvn clean install -Dversion.org.wildfly.openssl.natives=2.2.3.CR1-SNAPSHOT

Reply:
@fjuma Thank you, it appears to be working when I put it right under the build: https://github.com/Skyllarr/wildfly-openssl-natives/actions/runs/3687770319/jobs/6241763289 . Should I put a version in a parameter and add it to PR or this was just a test?

Reply:
Great! I don't see "wildfly-openssl" in the job output there but maybe I'm missing it? If you could add it to the PR with a version parameter that would be great. But don't worry about it if you don't have time.

Reply:
@fjuma Ah sorry, yes it seems to have ignred it for some reason, it did not run. I just missed it